students/rod_musser/lesson08/circle.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+from math import pi
+from functools import total_ordering
+
+"""
+A class representing a circle
+"""
+
+
+@total_ordering
+class Circle(object):
```

Comment:
note that in python3 you don't need to extend from object, you can simply do `class Circle:`